### PR TITLE
Patch: Add unit tests

### LINKED ISSUE #35: Write unit tests for utility functions
We have several functions for data formatting, address formatting, etc. Might be good to add some unit tests.

### PR DESCRIPTION
**Pull request by ECWireless**
Closes #35

### CODE REVIEW

File: src/utils/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  it('should format string relays', () => {
+    const relays = formatRelays('100000000');
+    expect(relays).toEqual('100.00');
+  });
+
+  it('should format days left as a percentage', () => {
+    const fillPercentage = formatFillPercentage(500, 5000);
+    expect(fillPercentage).toEqual(90);
```
Comment (Evalir) [inline src/utils/index.test.ts]: Could we just add a few more tests for these? Mostly for edge cases, like very big or very small numbers :)

**Review by Evalir — APPROVED** [review]
LGTM! Just generally asking for a few more tests :)

File: src/utils/index.test.ts (hunk 1 above)
Comment (ECWireless) [reply src/utils/index.test.ts]: Fixed in 369bea50965c69257b96988549e13de5abb02769

**Review by Evalir — APPROVED** [review]
Looking good—unless I'm missing anything, let's merge!
